app/graphql/queries/add_npo.rb

```diff
@@ -0,0 +1,10 @@
+module Queries
+    class AddNpo < Queries::BaseQuery
```

Comment:
Can you elaborate on why this query is called addNpo? I would think it would be called getNpos or searchNpos but maybe I'm misunderstanding what the code is doing

Reply:
I also agree with what Logan said

Reply:
🤷🏽

Reply:
I just called it that and ran with it lol. I was thinking that we are adding it to the recipe I guess. Totally happy to change it!

Reply:
Cool I was just wondering. Thanks for changing it though!